Code/Tools/AssetProcessor/native/ui/ProductAssetDetailsPanel.h

```diff
@@ -110,5 +116,10 @@ namespace AssetProcessor
         QListWidget* m_missingDependencyScanResults = nullptr;
         // The asset database connection in the AzToolsFramework namespace is read only. The AssetProcessor connection allows writing.
         AZStd::shared_ptr<AssetDatabaseConnection> m_assetDatabaseConnection;
+
+        ProductDependencyTreeModel* m_outgoingDependencyTreeModel = nullptr;
+        ProductDependencyTreeModel* m_incomingDependencyTreeModel = nullptr;
+        ProductDependencyTreeDelegate* m_outgoingDependencyTreeDelegate = nullptr;
+        ProductDependencyTreeDelegate* m_incomingDependencyTreeDelegate = nullptr;
```

Comment:
are you using these if not its safe to not store the pointers.

Reply:
They are used by `{Outgoing|Incoming}ProductDependenciesTreeView->setItemDelegate()` in [ProductAssetDetailsPanel.cpp](https://github.com/o3de/o3de/blob/50ef63f662cb1024c10c9580e45b36f6c149ec69/Code/Tools/AssetProcessor/native/ui/ProductAssetDetailsPanel.cpp#L57). When you say not store the pointers, I believe you mean simply create the delegate and pass pointers into `setItemDelegate()`, and not store them in this panel object. Is that correct?

Reply:
yea

Reply:
Does it mean that pointers of `m_{outgoing|incoming}DependencyTreeModel` can/should also be removed from the panel object? They are only used in `{Outgoing|Incoming}ProductDependenciesTreeView->setModel()` and connecting signal-and-slot, so model reacts when selected product asset itemchanges.